.gitlab/source_test/linux.yml

```diff
@@ -148,6 +148,16 @@ lint_flavor_heroku_linux-x64:
   variables:
     FLAVORS: '--flavors heroku'
 
+lint_python:
+  stage: source_test
+  image: gcr.io/datadoghq/agent-circleci-runner:v31988376-bfbb3afb
```

Comment:
Not for this PR but I think we will be able to switch to the another buildimage later and drop the circleci one from the buildimage repo. What do you think?

Reply:
I fully agree. We might want to select which image would be the "basic" one